### PR TITLE
[WIC] New HLSP blog

### PR DESCRIPTION
**Pull request by amcody**

### CODE REVIEW

File: content/news/2017-HLSPs.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Title: New and Updated K2 High Level Science Products
+Date: 2017-08-16 12:00
+Author: Ann Marie Cody
+
+The K2 communinity has been very active in working with the mission's images and light curves, and a number of groups have made
```
Comment (mirca) [inline content/news/2017-HLSPs.md]: ``communinity`` ---> ``community``

File: content/news/2017-HLSPs.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+Title: New and Updated K2 High Level Science Products
+Date: 2017-08-16 12:00
+Author: Ann Marie Cody
+
+The K2 communinity has been very active in working with the mission's images and light curves, and a number of groups have made 
+their output available to the public in the form of High Level Science Products. We are happy to report updates to several of these 
+submissions over the last 6 months. The K2 Mission encourages further submissions of HLSPs and our Guest Observer Office is happy 
+to help astronomers interface with the MAST archive.
```
Comment (barentsen) [inline content/news/2017-HLSPs.md]: "is happy" => "is available"  (NASA English)

File: content/news/2017-HLSPs.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+This month, the [K2 Systematics Correction](https://archive.stsci.edu/prepds/k2sc/) ("K2SC") by Aigrain et al. is celebrating the 
+release of version 2, with light curves now available through Campaign XX. These data products provide time series detrended for
```
Comment (barentsen) [inline content/news/2017-HLSPs.md]: "is celebrating the" => "has made available the" (NASA English)

File: content/news/2017-HLSPs.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+Author: Ann Marie Cody
+
+The K2 communinity has been very active in working with the mission's images and light curves, and a number of groups have made 
+their output available to the public in the form of High Level Science Products. We are happy to report updates to several of these 
+submissions over the last 6 months. The K2 Mission encourages further submissions of HLSPs and our Guest Observer Office is happy 
+to help astronomers interface with the MAST archive.
+
+This month, the [K2 Systematics Correction](https://archive.stsci.edu/prepds/k2sc/) ("K2SC") by Aigrain et al. is celebrating the 
+release of version 2, with light curves now available through Campaign XX. These data products provide time series detrended for 
+positional jitter and/or time-dependent systematics. As such, they are well suited to studying variable stars.
```
Comment (barentsen) [inline content/news/2017-HLSPs.md]: "they are well suited to" => "they are intended to be well suited to" (NASA English)

File: content/news/2017-HLSPs.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+Campaign 11 light curves.
+
+[POLAR](https://archive.stsci.edu/prepds/polar/), a new High Level Science Product was made available in March by Barros et al. 
+POLAR consists of K2 light curves derived from photometric apertures optimized for bright stars. The light curves have
+been detrended for position-dependent systematic effects, and the group has provided a list of possible planet transits
+selected from this data. 
+
+Finally, in February, [version 2.0 of EVEREST](https://archive.stsci.edu/prepds/everest/) was released, including Campaigns 0 
+through 8. The update now, detailed in Luger et al. (2017), includes light curves for short cadence and saturated targets. 
+Precision has improved over version 1.0, with photometric performance now equivalent to that of the Kepler main mission for stars
```
Comment (barentsen) [inline content/news/2017-HLSPs.md]: "now equivalent to" => "claimed to be equivalent to"  (NASA English)

File: content/news/2017-HLSPs.md (hunk 3 above)
Comment (barentsen) [inline content/news/2017-HLSPs.md]: You can add the following:

Version 2.0 fixes the "cadence" and "quality" columns in the FITS files to be 32-bit integers. The previous Version 1.0 had these defined as 16-bit integers, thus making them not usable. Version 2.0 includes a re-processing of Campaigns 3-6, and also adds Campaigns 7, 8, and 10 for the first time.

File: content/news/2017-HLSPs.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+[POLAR](https://archive.stsci.edu/prepds/polar/), a new High Level Science Product was made available in March by Barros et al. 
+POLAR consists of K2 light curves derived from photometric apertures optimized for bright stars. The light curves have
+been detrended for position-dependent systematic effects, and the group has provided a list of possible planet transits
+selected from this data. 
+
+Finally, in February, [version 2.0 of EVEREST](https://archive.stsci.edu/prepds/everest/) was released, including Campaigns 0 
+through 8. The update now, detailed in Luger et al. (2017), includes light curves for short cadence and saturated targets. 
+Precision has improved over version 1.0, with photometric performance now equivalent to that of the Kepler main mission for stars 
+down to Kp=14.
```
Comment (barentsen) [inline content/news/2017-HLSPs.md]: Maybe add a paragraph at the end explaining that more detailed documentation is available on each of the HLSP's pages at MAST?

**Comment by amcody** [comment]
Changes made- check it out =)

**Comment by barentsen** [comment]
Thanks again @amcody!  I took the liberty to edit this blog post to make sure all four teams are presented in a similar way, and include a mention of the large program value-added product requirement.

@gully Because you have been looking into community pipelines as well, would you mind giving this blog post a quick read to make sure we represent everyone fairly?  This should be good to merge if @gully gives the green light!

**Comment by gully** [comment]
:shipit:

**Comment by barentsen** [comment]
Thanks @gully.  I'm going to merge and publish this!

**Pull request merged**